src/Service/Event/EventService.php

```diff
@@ -98,4 +98,50 @@ public function checkIsEventDateExist(\DateTimeInterface $dateTime): bool
 
         return false;
     }
+
+    public function groupEventByTimeInterval(array $events, \DateTimeInterface $today = null): array
+    {
+        if (null === $today) {
+            $today = new \DateTime('today');
+        }
+
+        return [
+            'active' => \array_filter($events, function (Event $event) use ($today) {
```

Comment:
I would not use an anonymous function here, can you please refactor to a private method